packages/framework/pr_tools/PullRequestLinuxDriver.sh

```diff
@@ -8,6 +8,16 @@ source ${SCRIPTPATH:?}/common.bash
 on_weaver=$(echo "$@" | grep '\-\-on_weaver' &> /dev/null && echo "1")
 on_ats2=$(echo "$@" | grep '\-\-on_ats2' &> /dev/null && echo "1")
 
+# Configure ccache via environment variables
+function configure_ccache() {
+    print_banner "Configuring ccache"
+
+    envvar_set_or_create CCACHE_NODISABLE true
+    envvar_set_or_create CCACHE_BASEDIR '/fgs/trilinos/ccache/cache'
```

Comment:
Sorry I missed this before, but I think what we want is this:
CCACHE_DIR '/fgs/trilinos/ccache/cache'
CCACHE_BASEDIR '/path/to/trilinos/source/code'

I imagine that second one is available as a variable in here somewhere.